### ISSUE TITLE
No iOS15 support?

### ISSUE THREAD

**Issue by BenWoodford** [googlevr/cardboard#337]
Does this plugin actually work on iOS 15 currently? I notice all the releases are tested against 14, and if I switch into VR mode (which works absolutely fine on Android) in my Unity app I just get a black screen, followed by a view with no gyro if I minimise and re-open the app without killing it - and the corner buttons seem to be very difficult to press.

**Comment by johnfriedrich** [comment]
Hi, I was testing the demo scene of sdk version 1.12 on iOS 15.3 and iOs 15.4 beta just fine lately. Unity 2020.3.25. Maybe there is something wrong on your side?

**Comment by rubit0** [comment]
Works for me on iOS 15.1 build from Unity 2020.3. Maybe you forgot to enable the Cardboard XR Plugin under the iOS section?

**Comment by BenWoodford** [comment]
The plugin is definitely enabled. If I minimise the app and re-open it the masked eye view shows up but without gyro input - and that is definitely a CardboardXR thing

**Comment by rubit0** [comment]
Yeah I remember when I integrated into my project, that head movement was not working but I'm not sure anymore what I did. In worst case I could share my project with you.

But on other hand you can actually forget about all of this, I just noticed that on iOS, the right Eye camera only runs with half the framerate which makes it a uncomfortably dizzying experience and practically unusable.

**Comment by BenWoodford** [comment]
Interesting... @johnfriedrich do you get the same issue?

**Comment by BenWoodford** [comment]
So... it seems in my error I didn't realise that this is actually behaving identically on Android, which poses a lot of new questions.

**Comment by rubit0** [comment]
I should mention that I'm using version 1.11.0 of the Cardboard SDK, so maybe try a downgrade to that version (I assume you are on 1.12.0).

**Comment by BenWoodford** [comment]
I've tried every version from 1.6 through to 1.13 at this point. Hello Cardboard works but this doesn't. I'm going to revisit my code for switching to VR view though

**Comment by arilow** [comment]
Thanks @BenWoodford for reporting this issue! And sorry for the long delay. May I ask you what is the current status of this?


If it is still an issue, in order to allow us to help, could you please provide:
- The required steps to expose the issue.
- Is this behavior reproducible with the Hello Cardboard sample scene?
-  The Xcode version being used.
- Unity version.
- Is this issue still reproducible on Android?
- For this particular case, a video or a picture depicting the described behavior would also be very helpful.

Thanks!

**Comment by BenWoodford** [comment]
Not sure what fixed it, but _something_ has... if I ever work out what, I'll let you know

**Comment by arilow** [comment]
Closing as this seems to be fixed.


Thanks!
